plugins/api-docs/README.md

```diff
@@ -261,3 +261,5 @@ createApiFactory({
     },
   })
 ```
+
+In the same way as the `requestInterceptor` you can override any property of Swagger UI
```

Comment:
Also probably should be better replace the existing docs to show an actual example on how to use it with `swaggerUiProps`

Reply:
Not sure to understand the feedback 🤔 
There is an example on how to use the `swaggerUiProps` with the example of `requestInterceptor`